.github/workflows/maven-release.yml

```diff
@@ -10,36 +10,68 @@
 # governing permissions and limitations under the License.
 #
 
-name: Publish package to the Maven Central Repository
-on: workflow_dispatch
+name: Publish Release
+on: 
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'tag/version'
+        required: true        
 
+      action_tag:
+        description: 'Create tag? ("no" to skip)'
+        required: true
+        default: 'yes'
 jobs:
```

Comment:
Userprofile update some changes for its yml, now with no ref:main and added distribution: 'zulu'.  To make it consistent, see if you want to do the same for places.

jobs:
  publish:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3

      - name: Set up Java
        uses: actions/setup-java@v3
        with:
          distribution: 'zulu'
          java-version: 17

Reply:
Got that in